hack/kube-api-linter/exceptions.yaml

```diff
@@ -50,4 +50,34 @@
 - text: "field IngressPortStatus.Error has conflicting markers: optional_vs_required: {\\[optional\\], \\[kubebuilder:validation:Required\\]}. fields cannot be both optional and required"
   path: "staging/src/k8s.io/api/networking/v1beta1/types.go"
 
+# The following exceptions are for fields in stable or deprecated APIs that cannot be
+# changed due to backward compatibility constraints. Each rule is scoped to the
+# specific field to avoid hiding new violations.
 
+## For the "Extra" field, which is common across several auth-related APIs.
+- text: "field (UserInfo|SubjectAccessReviewSpec|CertificateSigningRequestSpec).Extra should not use a map type, use a list type with a unique name/identifier instead"
+  path: "staging/src/k8s.io/api/(authentication|authorization|certificates)/(v1|v1beta1)/types.go"
+
+## For ResourceList fields in the core API.
+- text: "field (PersistentVolumeSpec.Capacity|ContainerStatus.AllocatedResources|PodSpec.Overhead|ResourceQuotaSpec.Hard) type ResourceList should not use a map type, use a list type with a unique name/identifier instead"
+  path: "staging/src/k8s.io/api/core/v1/types.go"
+- text: "field (ResourceRequirements|VolumeResourceRequirements).(Limits|Requests) type ResourceList should not use a map type, use a list type with a unique name/identifier instead"
```

Comment:
For both ResourceRequirements and VolumeResourceRequirements, both Limits and Requests exist right?